src/utilities/createRouterResolve.spec.ts

```diff
@@ -29,5 +29,6 @@ test('throws an error if route with params cannot be matched', () => {
   const routerRoutes = createRoutes(routes)
   const resolve = createRouterResolve(routerRoutes)
 
+  // @ts-expect-error
```

Comment:
This was a positive thing I ran into. This test is testing that resolve throws and error when invalid parameters are passed. With removing the implementation types, typescript also errored.